lxqt-config-appearance/lxqtthemeconfig.ui

```diff
@@ -51,6 +51,13 @@
      </column>
     </widget>
    </item>
+   <item>
+    <widget class="QCheckBox" name="wallpaperOverride">
+     <property name="text">
+      <string>Override independently set wallper</string>
```

Comment:
I agree with @paulolieuthier. Variations:

Use theme's wallpaper
Use theme wallpaper
Wallpaper from the theme